codice/ddf
# Ddf 4267 fix list upload attribute overrides

#### What does this PR do?
Fix  attribute overrides for uploading directly to a list.
#### Who is reviewing it? 

@SmithJosh @alexaabrd @glenhein 
#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@beyelerb 
@vinamartin
#### How should this be tested?

Configure upload attribute overrides (e.g. title). Create a saved result list. Upload an item to that list and verify the attribute overrides take effect.
#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4267](https://codice.atlassian.net/browse/DDF-4267)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
